index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Mobile Page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    
+    <header class="first">
+        <div class="first-img"><img src="./images/Welcome.png" alt="welcome button at the top left"></div>
+        <div class="first-img" > <img src="./images/Union.png" alt="a union of three vectors"> </div>
```

Comment:
- [ ] Kindly add more paddings to your toolbar as represented in figma template:
your design: 
> ![image](https://user-images.githubusercontent.com/91486791/205142985-491541b5-beb1-4f0b-9bc2-4cceb6c33b02.png)
figma: 
> ![image](https://user-images.githubusercontent.com/91486791/205143036-516909d4-5fed-450e-aebb-79fe54c4e861.png)